cob_twist_controller/CMakeLists.txt

```diff
@@ -24,29 +24,29 @@ include_directories(include ${catkin_INCLUDE_DIRS} ${EIGEN_INCLUDE_DIRS} ${oroco
 
 
 ## Build
+set(SRC_C_DIR "src/constraint_solvers")
+set(SRC_CS_DIR "${SRC_C_DIR}/solvers")
+
 # Declare a cpp library
+add_library(damping_methods src/damping_methods/damping_base.cpp src/damping_methods/damping.cpp)
+add_library(constraint_solvers ${SRC_CS_DIR}/constraint_solver_base.cpp ${SRC_CS_DIR}/unconstraint_solver.cpp ${SRC_CS_DIR}/wln_joint_limit_avoidance_solver.cpp ${SRC_CS_DIR}/weighted_least_norm_solver.cpp ${SRC_C_DIR}/constraint_solver_factory_builder.cpp)
+add_library(limiters src/limiters/limiter_base.cpp src/limiters/limiter.cpp)
+target_link_libraries(constraint_solvers damping_methods)
+
 add_library(augmented_solver src/augmented_solver.cpp)
-target_link_libraries(augmented_solver ${orocos_kdl_LIBRARIES})
+target_link_libraries(augmented_solver constraint_solvers ${orocos_kdl_LIBRARIES})
 
 add_library(${PROJECT_NAME} src/cob_twist_controller.cpp )
```

Comment:
somehow it does not fit anymore calling this library ```${PROJECT_NAME}``` resolving to ```cob_twist_controller``` as other libraries are called limiters and such....can we replace ```${PROJECT_NAME}``` e.g. with ```twist_control```in the rest of the CMakeLists.txt?